### ISSUE TITLE
Defect dojo throwing the 500 error showing “well this was unexpected

### ISSUE THREAD

**Issue by Nsai1997** [DefectDojo/django-DefectDojo#10328]
1.Installed the defect dojo over the docker in the Vm in the GCP
2.port forwarded and mapped ip to the external up and forwarded the host to that ip
3.Allowed in firewall rules as well
4.After running the defect dojo using the docker I am getting the above error I.e; error 500 

well this was unexpected

even tried the solution provided over the internet 

Tried with the clean start and removed the volumes. Restarted the docker but still the same issue

could you please help me with the above issue

**Comment by manuel-sommer** [comment]
Hi @Nsai1997, could you please ask in the OWASP Slack channel this question or use the official support. This is not an issue. 
@mtesauro I guess this can be closed.

**Comment by manuel-sommer** [comment]
Did you wait until the initializer was done?

**Comment by Nsai1997** [comment]
Yes after doing that I am able to access the dashboard but the. Problem here is 

API configuration is saying we have access to projects 

engagement is also showing you have access to the projects

but I am not able to see the vulnerabilities in the product from the sonarqube

I have given the correct project key as well.

**Comment by Nsai1997** [comment]
Hi team, when click on import vulnerabilities while i am importing the vulnerabilites from the sonarqube to defect dojo I am getting the below error.  An exception error occurred during the report import: api_scan_configuration of type <class 'dojo.models.Product_API_Scan_Configuration'> is not one of [<class 'dojo.models.Tool_Configuration'>] Here are all the options submitted: None

**Comment by manuel-sommer** [comment]
Hi @Nsai1997 , is the issue still relevant?
Otherwise, please open up a new ticket with addititonal description, e.g. DefectDojo version. To me this issue looks stale @valentijnscholten
